client/verta/verta/_registry/modelversion.py

```diff
@@ -433,9 +433,9 @@ def log_environment(self, env):
 
         self._fetch_with_no_cache()
         self._msg.environment.CopyFrom(env._msg)
```

Comment:
Since we are using a separate message object, this line
```python
self._msg.environment.CopyFrom(env._msg)
```
I think can be removed

Reply:
yes, thanks